configs/microblaze-zephyr-eabi.config

```diff
@@ -0,0 +1,63 @@
+CT_CONFIG_VERSION="3"
+CT_EXPERIMENTAL=y
+# CT_PREFIX_DIR_RO is not set
+# CT_LOG_PROGRESS_BAR is not set
+CT_ARCH_MICROBLAZE=y
+CT_MULTILIB=y
+CT_TARGET_CFLAGS="-G0 -fno-pic -mlittle-endian"
+# TODO: Does CT_TARGET_LDFLAGS exist?
+CT_TARGET_LDFLAGS="-mlittle-endian"
```

Comment:
`-mlittle-endian` should not be necessary if you build `microblazeel-zephyr-elf` toolchain.